### ISSUE TITLE
Canvas Misbehaving in Production Build

### ISSUE THREAD

**Issue by DhairyaBahl** [hash-define-organization/Hash-Whiteboard#2]
### Description

Canvas is misbehaving in production build. Not sure but I believe the problem is because of babel's transcompilation.

### How to achieve ?

Change the `NODE_ENV` and `BABEL_ENV` in `scripts/build.js` from development to production and then do `react build`. You will see how canvas is misbehaving.

**Comment by ShivanshCharak** [comment]
if this issue is free to work I would like work on this issue under Gssoc

**Comment by DhairyaBahl** [comment]
@ShivanshCharak Yes Ofcourse ! You can work on this issue ! Make sure to update PR status on daily basis over here ! Otherwise This will be assigned to someone else.

**Comment by ShivanshCharak** [comment]
can u make it clear how is it misbehaving for me the canvas is technically working fine after changing the node_env and babel_env to production. react build is a command?
